config.go

```diff
@@ -278,6 +278,12 @@ type Config struct {
 	// Optional: defaults to false
 	AutomaticCutover bool
 
+	// These two values configure the amount of times Ferry should attempt to
```

Comment:
What are the defaults, and where are they set? And what happens if they're not set?

Reply:
We need to default to `1` as a value of `0` retries will retry indefinitely.

Reply:
This is defaulted to `1` below in the `ValidateConfig` method